shapes.py

```diff
@@ -0,0 +1,29 @@
+class Shape:
+    pass
+
+class TShape:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
```

Comment:
Why do we have `x`, `y` here we already have position information in square tuples. What will be the information stores in these `x` and `y`?